apimodels/agent_models.go

```diff
@@ -58,3 +58,35 @@ type NextTaskResponse struct {
 type EndTaskResponse struct {
 	ShouldExit bool `json:"should_exit,omitempty"`
 }
+
+type CreateHost struct {
```

Comment:
This struct should have json tags as well

Reply:
Done